# HG changeset patch
# User Keir Fraser <keir.fraser@citrix.com>
# Date 1219920254 -3600
# Node ID e92dedc72471c02c8d6b05b74afed0d32ae86171
# Parent 18a18a7dec2b6ad8eabb3281a1535e8bec05d8d5
tools/xenmon/Makefile: Move LDFLAGS after $<

gcc expects libraries needed for object files to be specified after
the object. Linking usually does not fail, unless it is optimized (for
instance, using -Wl,-as-needed).

The related Gentoo bug is 135145 [ https://bugs.gentoo.org/135145 ].

From: Robert Buchholz <rbu@gentoo.org>
Signed-off-by: Keir Fraser <keir.fraser@citrix.com>
xen-unstable changeset:   18386:ab8212bd5604f58635e2b8bc6ffec6eddd211cf9
xen-unstable date:        Wed Aug 27 15:02:38 2008 +0100

--- a/tools/xenmon/Makefile	Thu Aug 28 11:43:40 2008 +0100
+++ b/tools/xenmon/Makefile	Thu Aug 28 11:44:14 2008 +0100
@@ -42,6 +42,6 @@ clean:
 
 
 %: %.c Makefile
-	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $<
+	$(CC) $(CFLAGS) $< $(LDFLAGS) -o $@
 xentrace_%: %.c Makefile
-	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $<
+	$(CC) $(CFLAGS) $< $(LDFLAGS) -o $@

